README.md

```diff
@@ -16,7 +16,7 @@ so your PR will get merged faster, and you can start enjoying your shiny new ent
 
 - Each entity definition file must live inside its own folder and both must follow the filename format explained below. 
 - The definition MUST be a valid YAML file.
-- The definition must contain at least the top-level fields `domain` and `type`, along with the fields `name` and `identifier`, located under `synthesis`. 
+- The definition must contain at least the top-level fields `domain` and `type`, along with the fields `name`, `identifier` and `conditions`, located under `synthesis`. 
```

Comment:
Should we rephrase this?

If I'm not wrong is a valid configuration to NOT have synthesis section. right?

For example a domain-type that is not being synthetized using our system but that has golden metrics.

Reply:
It is but I'm not sure if we want to elaborate on that? I'm afraid it might be more confusing for outsider contributors, although maybe we could just say something like `The synthesis section is optional only if you can ensure that the telemetry is being stamped with the entity guid and tags based on rules defined internally at NewRelic` (kinda like [this](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/example-entity-definition.yml#L6)), wdyt?

Reply:
You are right. Whichever you prefer. If we are afraid of confusing contributors we may just leave that detail out. 

I was just thinking that you can open a file now and see that it has no synth section and that may be also confussing.

Reply:
You are right, I ended up adding it to the docs, so if anyone's curious at least the information is there.